ios/brave-ios/App/Client.xcodeproj/xcshareddata/xcschemes/Debug (AppStore).xcscheme

```diff
@@ -109,15 +109,15 @@
          </CommandLineArgument>
          <CommandLineArgument
             argument = "-BRSkipOnboarding YES"
-            isEnabled = "YES">
+            isEnabled = "NO">
```

Comment:
You can remove these?

Reply:
Missed while testing, thx for the catch fix at https://github.com/brave/brave-core/pull/22800/commits/49e062b086e60f8faf9ab30529d76ba879888445